### ISSUE TITLE
Please add the function to hide links that contain certain characters or text.

### ISSUE THREAD

**Issue by hamoumiKen** [richardfrost/AdvancedProfanityFilter#480]
I would like the ability to not only hide offensive text, but to hide the links themselves that contain that text. I want to eliminate from my brain as much as possible the existence of sites that contain offensive text.

**Comment by tonyram57** [comment]
I agree but I don't think it's possible. It would either break the link or it's not possible in the browser.

**Comment by richardfrost** [comment]
Could you give a more detailed example of what you're thinking @hamoumiKen? I think that tonyram57 is right that changing anything with the actual link will result in a broken link, and I don't want to add that as a "feature", because it will likely lead to a bad experience for people.
